README.md

```diff
@@ -97,15 +97,16 @@ Delete item (file or folder)
 oneDriveAPI.items.delete({
   accessToken: accessToken,
   itemId: createdFolder.id
-}).then(() => {
+}).then((isDeleted) => {
```

Comment:
Can `isDeleted` ever be false? If we don't manage to delete a file, won't this throw an error?

Reply:
Yes, as the document and testing on Graph Explorer, It will throw 400 error response.
![image](https://user-images.githubusercontent.com/29449869/97939031-94f06280-1db5-11eb-9bea-94cbe2c483b2.png)

I think with this we will have to force the user to use try-catch because the error response from the request might not be 400 (not deleted) but other errors. We can get the error code and message via the `error` (`error.response.statusCode`) object in the `catch()`.

![image](https://user-images.githubusercontent.com/29449869/97947432-d128c080-1dbf-11eb-922f-d9aa8d3259ca.png)

About the response `boolean`, I now also think `isDeleted` can't be `false` so making it return `Promise<void>` like before is also OK if you want.

Reply:
if `isDeleted` is always true, then I would prefer that we are returning `Promise<void>` as it would be less confusing for users of the module. Try catch makes sense for me, that is how I would expect the function to behave 👍